### ISSUE TITLE
[WHL] IPC timeout in some cases

### ISSUE THREAD

**Issue by libinyang** [thesofproject/linux#801]
On WHL, there are some timeouts in initialization and in playback operation after apply the patch https://github.com/thesofproject/linux/pull/798

kernel: sof-dev: b5ece71

**Comment by wenqingfu** [comment]
The issue report is done badly. Can you please close and open a new one at least with dmesg/logger? If all you have is "I have some issue", what's the value of opening an issue and attracting others' attention?

**Comment by libinyang** [comment]
@wenqingfu When I firstly boot up, I met the timeout issue in initialization. And I found the playback is timeout. At first, I think they are the same issue. So I reported the issue contains 2 issues actually. And after Bard debugged, he found that they are different bugs. So I will file a bug for the playback timeout later. And I will paste the log when I report the issue. The other initialization timeout bug is very hard to reproduce.

> The issue report is done badly. Can you please close and open a new one at least with dmesg/logger? If all you have is "I have some issue", what's the value of opening an issue and attracting others' attention?
